src/main/java/com/tabnine/binary/requests/config/StateRequest.kt

```diff
@@ -0,0 +1,23 @@
+package com.tabnine.binary.requests.config
+
+import com.tabnine.binary.BinaryRequest
+import com.tabnine.binary.exceptions.TabNineInvalidResponseException
+import java.util.*
+
+class StateRequest : BinaryRequest<StateResponse> {
+    override fun response(): Class<StateResponse> {
+        return StateResponse::class.java
+    }
+
+    override fun serialize(): Any {
+        return Collections.singletonMap("State", Any())
```

Comment:
Kotlin got `mapOf` which is awesome: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/map-of.html

Reply:
And I've got to say I'm not sure what `Any()` does or if it identical to an (seems it is for this use-case from a short read online). Cool, learned something.

Reply:
From what I understand, `Any()` is the equivalent of Java's `new Object()`.
What is the alternative?